recipe/meta.yaml

```diff
@@ -24,15 +24,15 @@ requirements:
     - setuptools
     - toolchain
     - clangdev
-    - libcxx 4.0.*  # [osx]
-    - openmp 4.0.*  # [osx]
+    - libcxx >=4.0  # [osx]
+    - openmp >=4.0  # [osx]
```

Comment:
You should build with `4.0.*` and let the run requirements be `>=4.0`. Otherwise this package will be built with `5.0`, but at runtime 4.0 satisfies the constraints

Reply:
Good catch! Does package built with 4.0 run ok with 5.0?

Reply:
@aldanor, yes, that was intentional. openmp and libcxx got updated to version 5 and that causes xgboost to downgrade when other packages are installed.

Reply:
I also do not understand the current status of these libs and what conda-forge wants to be standard.

Reply:
@beckermr let me try

Reply:
> Does package built with 4.0 run ok with 5.0?

I've no idea. If you are not sure, then have `5.0.*` in both build an run requirements

Reply:
@beckermr, @isuruf it does work when built with 4.0 and run with 5.0.